enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -81,3 +84,17 @@ def _get_course_overview(self, course_run_id):
                 return overview
 
         return None
+
+    def _get_course_run_url(self, request, course_run_id):
+        """
+        Get the appropriate course url while incorporating Executive Education content.
+        """
+        course_key = CourseKey.from_string(course_run_id)
+        course_key_str = '{}+{}'.format(course_key.org, course_key.course)
+        course_data = get_course_catalog_api_service_client().get_course_details(course_key_str)
+        if course_data:
+            course_type = course_data.get("course_type")
+            if course_type == EXEC_ED_COURSE_TYPE:
+                return request.build_absolute_uri(EXEC_ED_LANDING_PAGE)
```

Comment:
I am not sure if we need `build_absolute_uri`? We already have absolute url.

Reply:
build_absolute_uri will ensure that any future updates to this config are 'absolute'